spec/spec_helper.rb

```diff
@@ -1,6 +1,14 @@
 #This file is generated by ModuleSync, do not edit.
 require 'puppetlabs_spec_helper/module_spec_helper'
 
+if Puppet::Util::Package.versioncmp(Puppet.version, '4.5.0') >= 0
+  RSpec.configure do |c|
+    c.before :each do
+      Puppet.settings[:strict] = :error
```

Comment:
Apt throws deprecation warnings and this means they are thrown as errors during testing, the current reason Travis run is failing.